### DISCUSSION TITLE
Results Thread (July 2023)

### DISCUSSION THREAD

**Discussion by NikolajHansen23** [NikolajHansen23/netreach#1] [General]
Please post your results, attach the detailed result (found in the output folder), and mention your location and ISP.

You could use the below template:

:checkered_flag:: 41.96%
:satellite:: MTN
:earth_africa:: Tehran, Iran
:mag:: [scan.txt](https://github.com/NikolajHansen23/netreach/files/12062737/scan.txt)

**Comment by NikolajHansen23** [comment]
🏁: 50.94%
📡: MCI
🌍: Tehran, Iran
🔍: [scan.txt](https://github.com/NikolajHansen23/netreach/files/12062761/scan.txt)

**Comment by NikolajHansen23** [comment]
🏁: 55.63%
📡: Arvan Cloud VPS
🌍: Tehran, Iran
🔍: [scan.txt](https://github.com/NikolajHansen23/netreach/files/12068724/scan.txt)

**Comment by ghost** [comment]
🏁: 28.75%😐
📡: MTN
🌍: Gilan, Iran
🔍: [scan.txt](https://github.com/NikolajHansen23/netreach/files/12073224/scan.txt)

Zare Poor and Dahaka i have a message for you: You're a piece of $hit.

**Comment by mkd2380** [comment]
🏁: 27.38%  💩
📡: VDSL,TCI
🌍: YZD, Iran
🔍:[scan-1689723075347.txt](https://github.com/NikolajHansen23/netreach/files/12089698/scan-1689723075347.txt)

**Comment by siq0o** [comment]
🏁: 63.62%
📡: AST
🌍: Tehran, Iran
🔍: [scan-1689720184961.txt](https://github.com/NikolajHansen23/netreach/files/12089699/scan-1689720184961.txt)
*******
🏁: 59.38%
📡: MTN
🌍: Tehran, Iran
🔍: [scan-1689723064324.txt](https://github.com/NikolajHansen23/netreach/files/12089704/scan-1689723064324.txt)
*******
Was able to only achieve 77.43% with no restrictions, using cloudflare over v2ray (10 megabits per second down speed) :
[scan-1689719407490.txt](https://github.com/NikolajHansen23/netreach/files/12089718/scan-1689719407490.txt)
*******
Update : seems to be unreliable running with the default settings
96.30% using a wireguard connection
[scan-1689725960479.txt](https://github.com/NikolajHansen23/netreach/files/12089864/scan-1689725960479.txt)

**Reply by NikolajHansen23** [reply]
Most of the time, I can't have stable (i.e., "jitterless") connections using configs that use Cloudflare's CDN. That may be it.

However, I'm going through the websites list one more time to remove flaky websites. Will let you know to do the test again then.

**Reply by NikolajHansen23** [reply]
Updated the websites. You could give it another shot now.

**Comment by ghost** [comment]
> Most of the time, I can't have stable (i.e., "jitterless") connections using configs that use Cloudflare's CDN. That may be it.
> 
> However, I'm going through the websites list one more time to remove flaky websites. Will let you know to do the test again then.

I used to connect by free vmess websocket accounts + cloudflare worker until they blocked cloudflare's IPs.
There was an annoying problem,
Some random websites wouldn't load every time i connected with those configs. chrome would show me "no connection" errors immediately when i clicked enter (mostly on twitter and youtube) but other sites would open without any error.
I could fix it temporarily by connecting with other configs or restarting the connection.
Do you know what was the problem?
I'm curious to know.

**Reply by NikolajHansen23** [reply]
I can't say for sure. However, some websites are banned by the VPN admin. It could also be that the VPN IP has been blocked by some provider (Google Cloud blocked my IP once). It could also be traffic congestion since it was a free VPN.

**Comment by NikolajHansen23** [comment]
🏁: 98.43%
📡: Hetzner VPS
🌍: Nuremberg, Germany
🔍: [scan.txt](https://github.com/NikolajHansen23/netreach/files/12112741/scan-1689874468843.txt)

**Comment by ghost** [comment]
🏁: 56.66%
📡: Shatel
🌍: Gilan, Iran
🔍: [scan.txt](https://github.com/NikolajHansen23/netreach/files/12139859/scan.txt)
